src/github/mod.rs

```diff
@@ -195,6 +202,55 @@ impl SyncGitHub {
         Ok(diffs)
     }
 
+    // collect all org and respective teams members in a HashMap
+    fn map_teams_to_org(&self) -> anyhow::Result<HashMap<String, HashSet<u64>>> {
+        let mut org_team_members: HashMap<String, HashSet<u64>> = HashMap::new();
+
+        for team in &self.teams {
+            let mut team_org;
+
+            if let Some(gh) = &team.github {
+                for toml_gh_team in &gh.teams {
+                    team_org = toml_gh_team.org.clone();
```

Comment:
```suggestion
                    let team_org = toml_gh_team.org.clone();
```
No need to have the variable in the external scope now :)